rllib/algorithms/bc/bc.py

```diff
@@ -98,6 +98,7 @@ def build_learner_connector(
         # Remove unneeded connectors from the MARWIL connector pipeline.
         pipeline.remove("AddOneTsToEpisodesAndTruncate")
         pipeline.remove("GeneralAdvantageEstimation")
+        pipeline.remove("NumpyToTensor")
```

Comment:
Aaahh! So this was one of the problems? That we were already converting everything to torch tensors?

Reply:
Yes it was one of them. The major one, was however, that we were passing in a learner that runs on GPU and that one needed to be serialized from ray to send it to the data workers. When deserializing it there it errored out.